Makefile

```diff
@@ -46,3 +49,24 @@ test-bindings: $(PKG_BINDINGS_DIR)
 
 clean-bindings:
 	rm -rf $(PKG_BINDINGS_DIR)
+
+image: Dockerfile
+	DOCKER_BUILDKIT=1 \
+	$(DOCKER) build \
+	    --progress=plain \
+	    --tag $(BUILDIMAGE) \
+	    .
+
+docker-%: image
```

Comment:
Can you add specific targets for the `docker` variants so that they exist for tab-complete?

It should be as simple as changing this to:
```
TARGETS = all test clean bindings test-bindings clean-bindings
docker-$(TARGETS): image
	...
```
We can probably then change the .PHONY line above to just reference $(TARGETS) as well.

Reply:
Updated:
```
$ make\t\t
all                    clean                  docker-all             docker-clean           docker-test            image                  test-bindings
bindings               clean-bindings         docker-bindings        docker-clean-bindings  docker-test-bindings   test
```